demos/JWST/S5_fit_par_template.epf

```diff
@@ -19,30 +19,31 @@ rp           0.16          'free'         0.05         0.3          U
 # ------------------
 # ** Orbital parameters **
 # ------------------
-per          0.813473978   'free'         0.813473978  0.000000035  N
-t0           55528.353027  'free'         55528.34     55528.36     U
-time_offset  0             'independent'
-inc          82.109        'free'         82.109       0.088        N
-a            4.97          'free'         4.97         0.14         N
-ecc          0.0           'fixed'        0            1            U
-w            90.           'fixed'        0            180          U
+per          4.055259   	'free'        		4   		4.1 		U
+t0           0.53		 	'free'         0.35     0.60     U
+time_offset  0            	'independent'
+inc          87.93        	'free'         85       90        U
+a            11.612         'free'         10         13         U
+ecc          0.0           	'fixed'        0            1            U
+w            90.           	'fixed'        0            180          U
 # -------------------------
 # ** Limb darkening parameters **
 # Choose limb_dark from ['uniform', 'linear', 'quadratic', 'kipping2013', 'square-root', 'logarithmic', 'exponential', '4-parameter']
 # -------------------------
-limb_dark    'kipping2013' 'independent'
-u1           0.3           'free'         0            1            U
-u2           0.1           'free'         0            1            U
+limb_dark    'quadratic'	'independent'
```

Comment:
kipping2013 is really recommended over quadratic - it samples more efficiently and ensures physicality

Reply:
Sure, I think I just copied this .epf from a different test analysis I was doing, but can switch to Kipping 2013.